torch/fx/experimental/proxy_tensor.py

```diff
@@ -625,12 +625,7 @@ def wrap_fake_symbolic(x, sym_shape):
 
 
 def get_torch_dispatch_modes():
-    modes = [torch._C._get_torch_dispatch_mode()]
-    if modes[-1] is None:
-        return list()
-    while modes[-1].inner is not None:
-        modes.append(modes[-1].inner)
-    return modes
+    return torch.utils._python_dispatch._cur_torch_dispatch_mode + [torch._C._get_torch_dispatch_mode()]
```

Comment:
Huh, the list concat is unexpected, I would have expected torch.utils._python_dispatch._cur_torch_dispatch_mode to have the current mode too